content/environments.md

```diff
@@ -381,8 +381,7 @@ using the following methods:
 
 -   By using the `knife edit` and `knife exec` subcommands
 
--   By editing the `chef_environment` directly using knife or the Chef
-    management console
+-   By editing the `chef_environment` directly using knife
```

Comment:
I think this line can go away entirely since the line above it covers the same thing

Reply:
done